karman-avionics/src/utils/CC2500_regvalues.c

```diff
@@ -1,59 +0,0 @@
-/*****************************************
```

Comment:
See above comment... Autogenerated...

Reply:
This file is unnecessary until we go back to targeting the CC2500 radio module. For now, it's not needed.